### PR TITLE
Newsletter about LAL workshop: initial draft

### PR DESCRIPTION
**Pull request by jouvin**
@HEP-SF/startup please read and comment, preferably on GitHub (in the pull request).

### CODE REVIEW

File: newsletter/_posts/2016-05-17-workshop-lal.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
+title: Second HSF Workshop
+author: Michel Jouvin
+layout: newsletter
+---
+
+# Second HSF Workshop (May 2-4, 2016)
+
+The HSF had its second workshop at LAL (Orsay, France) on May 2-4, 2016. It was well attended (70 people) representing many different laboratories, universities and institutions (~30) from both sides of the Atlantic, involved in different experiments (LHC, BELLE II, Intensity Frontier...). We were happy to see a lot of new faces! [Agenda](https://indico.cern.ch/event/496146/timetable/) was a mix of (lively!) general discussions and topical/hands-on sessions. Live notes of these discussions are [available](????). This newsletter summarizes them.
```
Comment (hegner) [inline newsletter/_posts/2016-05-17-workshop-lal.md]: I would change the numbering there. We had the kick-off one at CERN, the first follow up at SLAC, the second follow up at LAL.

File: newsletter/_posts/2016-05-17-workshop-lal.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+One of the breaking news is that the HSF now has a logo! Thanks to J. Lingermann, the winner of the logo contest, and to all of you who took the time to submit a logo proposal.
```
Comment (hegner) [inline newsletter/_posts/2016-05-17-workshop-lal.md]: Lingemann

File: newsletter/_posts/2016-05-17-workshop-lal.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+
+## Projects and Project Support
```
Comment (hegner) [inline newsletter/_posts/2016-05-17-workshop-lal.md]: I'd add a link to all of those.

File: newsletter/_posts/2016-05-17-workshop-lal.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+In addition, a new activity is starting: peer-reviews of software projects. The first review has been requested by GeantV and will take place next October.
+
+
+## Software Packaging
+
+It has been one of the topical discussions. Software build tools and packaging are a key piece of the software interoperability. As we cannot envision all the projects adopting the same tools, the effort has concentrated on a review of the existing tools and an attempt to identify the tool with the greater potential as the integrating layer. One tool from the HPC world, Spark, generated a lot of hype. Some people tried to experiment it in various contexts in the last months and the workshop has been the occasion of an in-depth discussion about Spark potential and limits and how to move forward.
```
Comment (hegner) [inline newsletter/_posts/2016-05-17-workshop-lal.md]: *spack* not Spark. ;-)

File: newsletter/_posts/2016-05-17-workshop-lal.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+See the [Packaging WG](http://hepsoftwarefoundation.org/workinggroups/2015/11/04/packaging.html) section on the web site for more details.
+
+
+## Learning from Others
+
+We got three interesting presentations from projects with goals similar to the HSF, in other context:
+* BioConductor: an open-source project portal associated with an ecosystem of tools for bioinformatics
+* Netherlands eScience Center (NeSC): a team of computing experts working with all the scientific fields, building a common expertise that can benefit to all of them
+* Deepsy: a project and platform to promote credit for software in science. Currently focused on Python and R ecosystems, it analyses software packages and sources to identify contributions and make visible people contributions even when this is made of "small contributions" to many projects.
```
Comment (hegner) [inline newsletter/_posts/2016-05-17-workshop-lal.md]: Depsy

**Comment by sextonkennedy** [comment]
I'm with the group of people that says this is the 3rd workshop.  It's true we mis-numbered the SLAC one but I don't think anyone will call us on it or even remember for that matter since it was almost 1.5 years ago.

**Pull request merged**